### PR TITLE
added checkbox option for BoolField for antd

### PR DESCRIPTION
**Pull request by purplecones**
PR adds ability to create an antd `Checkbox` with uniforms `BoolField` instead of just the antd `Switch`.

### Usage
```javascript
<BoolField
    checkbox
    name="myCheckbox"
    label="Check here if you agree"
/>
```

### CODE REVIEW

File: packages/uniforms-antd/__tests__/BoolField.js [hunk 1]
```diff
@@ -1,83 +1,155 @@
 import React   from 'react';
 import Switch  from 'antd/lib/switch';
+import Checkbox  from 'antd/lib/checkbox';
 import {mount} from 'enzyme';
```
Comment (radekmie) [inline packages/uniforms-antd/__tests__/BoolField.js]: These should be aligned.

File: packages/uniforms-antd/src/BoolField.js [hunk 2]
```diff
@@ -1,26 +1,38 @@
 import Icon           from 'antd/lib/icon';
 import React          from 'react';
 import Switch         from 'antd/lib/switch';
+import Checkbox       from 'antd/lib/checkbox';
 import connectField   from 'uniforms/connectField';
 import filterDOMProps from 'uniforms/filterDOMProps';
```
Comment (radekmie) [inline packages/uniforms-antd/src/BoolField.js]: These should stay in order.

File: packages/uniforms-antd/src/BoolField.js [hunk 3]
```diff
@@ -1,26 +1,38 @@
...
+            /> :
+            <Switch
+                checked={props.value}
+                disabled={props.disabled}
+                id={props.id}
+                name={props.name}
+                onChange={() => props.onChange(!props.value)}
+                ref={props.inputRef}
+                {...filterDOMProps(props)}
+            />
```
Comment (radekmie) [inline packages/uniforms-antd/src/BoolField.js]: How about `React.createElement(props.checkbox ? Checkbox : Switch, {...})`? It'll reduce this duplicated code.

File: packages/uniforms/src/filterDOMProps.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ const unwantedProps = [
     // These props are provided by BaseField
     'changed',
     'changedMap',
+    'checkbox',
```
Comment (radekmie) [inline packages/uniforms/src/filterDOMProps.js]: These are global, so it's a breaking change. I'd rather go with destructuring props in `BoolField` - it won't be needed then.

**Comment by purplecones** [comment]
Thanks for feedback, learning something new every day.

**Pull request merged**